### PR TITLE
Enable inactive deserialization for DDIC objects

### PR DESCRIPTION
**Pull request by wurzka**
Enabling inactive deserialization for DDIC objects in Common-AFF class

Adding DTEB to DDIC objects list

### CODE REVIEW

File: src/objects/aff/zcl_abapgit_object_common_aff.clas.abap [hunk 1]
```diff
@@ -310,6 +310,16 @@ CLASS zcl_abapgit_object_common_aff IMPLEMENTATION.
           ENDIF.
         ENDLOOP.
 
+        IF is_active( ) = abap_false.
+          " as DDIC-object e.g. are not deserialized in active state, activation must be performed
+          CALL FUNCTION 'RS_INSERT_INTO_WORKING_AREA'
```
Comment (mbtools) [inline src/objects/aff/zcl_abapgit_object_common_aff.clas.abap:315]: `zcl_abapgit_objects_activation=>add_item( ms_item ).` will add the object to the AG mass activation queue

**Review by mbtools — CHANGES_REQUESTED** [review]

File: src/objects/aff/zcl_abapgit_object_common_aff.clas.abap (hunk 1 above)
Comment (wurzka) [reply src/objects/aff/zcl_abapgit_object_common_aff.clas.abap:315]: Only by add_item without RS_INSERT_INTO_WORKING_AREA the changes on DTEB were not correctly activated. Is there anything else needed for activation for DTEB?
When adding RS_INSERT_INTO_WORKING_AREA and opening a DTEB object in ADT editor after PULL, we see the object in inactive state with inserted changes and can activate it afterwards. Without RS_INSERT_INTO_WORKING_AREA, no changes are shown and object is still in active state without the changes expected by pull.

Comment (wurzka) [reply src/objects/aff/zcl_abapgit_object_common_aff.clas.abap:315]: Ah thanks! Then we have to check it again.

Comment (wurzka) [reply src/objects/aff/zcl_abapgit_object_common_aff.clas.abap]: We checked it. Removing RS_INSERT_INTO_WORKING_AREA and only performing add_items works for DTEB when its abapgit object handler redefines zif_abapgit_object~get_deserialize_steps like above

**Review by mbtools — APPROVED** [review]
LGTM 👍

**Pull request merged**